### ISSUE TITLE
New Clang warning: "'extern' variable has an initializer"

### ISSUE THREAD

**Issue by snej** [SRombauts/SQLiteCpp#94]
The latest change to Database.cpp triggers a compiler warning from Clang:
```
/Couchbase/SQForest/vendor/SQLiteCpp/src/Database.cpp:35:21: warning: 'extern' variable has an initializer [-Wextern-initializer]
extern const char*  VERSION        = SQLITE_VERSION;
```
The `extern` qualifiers on this and the other declarations here should be removed, since these are the actual definitions of the constants, not just references to them.

**Comment by SRombauts** [comment]
Hi, thanks for reporting!

**Comment by SRombauts** [comment]
Thanks to you, I released the v2.0.0 without a warning.
Cheers!
